__tests__/app.test.js

```diff
@@ -69,3 +69,40 @@ describe('GET /api/articles/:article_id', () => {
       })
   })
 })
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('status:200, responds with updated article object', () => {
+    const newVote = {
+      inc_votes: 5,
+    }
+    return request(app)
+      .patch('/api/articles/3')
+      .send(newVote)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body
+        expect(article).toEqual({
+          article_id: 3,
+          author: 'icellusedkars',
+          body: 'some gifs',
+          created_at: '2020-11-03T09:12:00.000Z',
+          title: 'Eight pug gifs that remind me of mitch',
+          topic: 'mitch',
+          votes: 5,
+        })
+      })
+  })
```

Comment:
We could also check to make sure that a negative vote (downvote) works too!

Reply:
Ok, thanks for comment.